fusor-ember-cli/app/controllers/openstack/register-nodes.js

```diff
@@ -101,7 +101,10 @@ const RegisterNodesController = Ember.Controller.extend(ProgressBarMixin, NeedsD
   }),
 
   nodeDriverHumanized: Ember.computed('nodeInfo.driver', function () {
-    return this.get('drivers').findBy('value', this.get('nodeInfo.driver')).label;
+    let driver = this.get('drivers').findBy('value', this.get('nodeInfo.driver'));
+    if (driver) {
+      return driver.label;
```

Comment:
This was be unnecessary in a real system. Using the mirage mocks, I got an error for method 'label' of undefined.